libpod/runtime.go

```diff
@@ -89,6 +89,11 @@ var (
 	// DefaultDetachKeys is the default keys sequence for detaching a
 	// container
 	DefaultDetachKeys = "ctrl-p,ctrl-q"
+
+	// minConmonMajor is the major version required for conmon
+	minConmonMajor = 2
+	// minConmonVersion is a string used to display the minimum conmon version
+	minConmonVersion = "v2.0.0"
```

Comment:
Can you make this
minConmonVersion = fmt.Sprintf("v%d.0.0", minConmonMajor)

Reply:
Or better yet remove this variable and just use the minConmonMajor in the error message below.

Reply:
fixed